pkg/apis/numaplane/v1alpha1/status_types.go

```diff
@@ -117,9 +117,37 @@ type ChildStatus struct {
 	AssessUntil *metav1.Time `json:"assessUntil,omitempty"`
 }
 
+// ScaleValues stores the desired min and max, scaleTo, and actual scale values of a pipeline or monovertext vertex
+type ScaleValues struct {
+	// DesiredMin is the min scale value of the original child spec
+	DesiredMin int64 `json:"desiredMin"`
+	// DesiredMax is the max scale value of the original child spec
+	DesiredMax int64 `json:"desiredMax"`
+	// ScaleTo indicates how many pods to scale down to and it is calculated by getting the floor of the desiredMax value divided by 2
```

Comment:
nit: even though the comment is helpful, I may prefer that we remove the part about how it's calculated from here, since conceivably it could be a different implementation for each rollout using this